src/App.js

```diff
@@ -0,0 +1,32 @@
+import './App.css';
+import Navbar from './components/Navbar/Navbar';
+import Button from './Asset/Button/Button';
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+const App = () => {
+  const [setUser] = useState({});
+  const [temp, settemp] = useState([]);
+  const getUser = async () => {
+    let randomNumber = Math.floor(Math.random() * 10);
```

Comment:
use const instead let

Reply:
also create one separate common util file at src level and add this method into that and then use it here